pipelines/orthofinder_modules.nf

```diff
@@ -125,34 +160,34 @@ process mergeRefToQuery {
 
     label 'lsf_default'
 
-    publishDir "${params.query_destination}/${query}"
+    publishDir "${params.query_destination}/${query}", mode: 'copy'
 
     input:
     val query
-    val query_fasta // todo: When DataFile API works this will be file not val
-    val ref_dir // todo: When DataFile API works this will be path not val
+    path query_fasta
+    val ref_dir
 
     output:
-    val "${query}"
+    path "${query}/"
 
     shell:
     """
-    mkdir ${query};
-    rsync -Lavz ${ref_dir} ${params.query_destination}/${query}/;
-    cp ${query_fasta} ${params.query_destination}/${query};
+    rsync -Lavz ${ref_dir}/ ${query};
+    cp ${query_fasta} ${query}/;
```

Comment:
Is there any sanity check  (task) after this process has been done? Or if this fails, the pipeline will stop anyway, isn't?

Reply:
There is going to be - there is a ticket for this to include CHECKSUMS etc